### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Temisolovn**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by Temisolovn — COMMENTED** [review]
This is a test on pull request comment.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (Temisolovn) [inline index.html:50]: This is a single comment. Great game

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
 <script>
```
Comment (Temisolovn) [inline index.html:52]: Comment

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (v-temisolovn) [inline index.html]:
```suggestion
This is a suggestion
```

**Pull request merged**